staging/src/k8s.io/csi-translation-lib/plugins/vsphere_volume_test.go

```diff
@@ -30,11 +30,11 @@ func TestTranslatevSphereInTreeStorageClassToCSI(t *testing.T) {
 	translator := NewvSphereCSITranslator()
 	topologySelectorTerm := v1.TopologySelectorTerm{[]v1.TopologySelectorLabelRequirement{
 		{
-			Key:    v1.LabelFailureDomainBetaZone,
+			Key:    v1.LabelTopologyZone,
```

Comment:
Can we have both beta and v1 unit tests?

Reply:
adding unit test for beta labels

```golang
	topologySelectorTermWithBetaLabels := v1.TopologySelectorTerm{[]v1.TopologySelectorLabelRequirement{
		{
			Key:    v1.LabelFailureDomainBetaZone,
			Values: []string{"zone-a"},
		},
		{
			Key:    v1.LabelFailureDomainBetaRegion,
			Values: []string{"region-a"},
		},
	}}
```

```golang
                {
			name:  "translate with storagepolicyname and allowedTopology beta labels",
			sc:    NewStorageClass(map[string]string{"storagepolicyname": "test-policy-name"}, []v1.TopologySelectorTerm{topologySelectorTerm}),
			expSc: NewStorageClass(map[string]string{"storagepolicyname": "test-policy-name", paramcsiMigration: "true"}, []v1.TopologySelectorTerm{topologySelectorTermWithBetaLabels}),
		},
```